lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -87,6 +87,27 @@ class AwsCompileEventBridgeEvents {
           required: ['inputTemplate'],
           additionalProperties: false,
         },
+        retryPolicy: {
+          type: 'object',
+          properties: {
+            maximumEventAge: {
+              type: 'integer',
+              minLength: 60,
+              maxLength: 86400,
+            },
+            maximumRetryAttempts: {
+              type: 'integer',
+              minLength: 0,
+              maxLength: 185,
+            },
+          },
+        },
+        deadLetterConfig: {
+          type: 'object',
+          properties: {
+            arn: { type: 'string' },
```

Comment:
Is there something that prevents us from also accepting that `arn` provided with CF intrinsic function?

Reply:
@pgrzesik hm, I'm not sure if I understood this one. Do you mean use `Fn::GetAtt` instead of passing the Arn manually?

Reply:
Yes, exactly, I think it might be a common pattern - I think nothing prevents us to supporting CF intrinsic functions here